index.d.ts

```diff
@@ -792,22 +810,22 @@ declare module 'fastest-validator' {
 		 * @param {any=} opts.actual
 		 * @param {MessagesType} opts.messages
 		 */
-		makeError(opts: { type: keyof MessagesType, field?: string, expected?: any, actual?: any, messages: MessagesType }): ValidationError;
+		makeError(opts: { type: keyof MessagesType, field?: string, expected?: any, actual?: any, messages: MessagesType }): string;
```

Comment:
Are you sure that the method should keep in the current definition? It looks like an internal method and shouldn't be exposed outside of current definitions

Reply:
It is used in [custom validator](https://github.com/icebob/fastest-validator#custom-validator) creation.

Reply:
Ok. That's weird, but I'm ok with it